src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
The stylesheet link references a `.scss` file, which is a Sass file. Ensure that this file is being processed into a `.css` file, and link to the resulting CSS file instead, as browsers do not directly interpret Sass files.